.github/workflows/gate.yaml

```diff
@@ -75,10 +75,10 @@ jobs:
         working-directory: ./build
 
   validate-fedora:
-    name: Build, Test on Fedora 33 (Container)
+    name: Build, Test on Fedora 34 (Container)
```

Comment:
When the job name changes the required test name in the project's settings need to be changed as well.
Should we remove the version from the name of the job?

Reply:
Sounds like a decent plan to me.

Reply:
It will make it less obvious which F was used to execute the test. I'm not insisting if people who work with the content closer than me are not against this change, just explaining my decision to include it there in the first place.

Reply:
NB: We could probably benefit from having a rolling Rawhide-based *optional* test to embrace things to come.

Reply:
Keeping it in the title is fine. Its only a once every month or year change. 

I have also added a raw hide test. However, that will fail since we not compatible with the new version of Jinja.